### DISCUSSION TITLE
p/d25cb9b2f5d446ce852b03c15e8fb8e5/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#11] [Announcements]
# p/d25cb9b2f5d446ce852b03c15e8fb8e5/

但弄雕虫之技，尘芥草莽，于泥泞自勉……  开发VSCode：基于 Web 技术最优秀的编辑器[便携版], [开源], [跨平台], [免费]  具体可以参考：官网, GitHub, GitBook 中文介绍  MS VSCode 是微软出品的一个跨平台，基于 Web 的，免费的开源文本编辑器。目前已经使用了半年多了，对其的了解也更加深入。就目前看来，VSCode 应该是使用 Web 技术实现的最

https://blog.rxliuli.com/p/d25cb9b2f5d446ce852b03c15e8fb8e5/

**Comment by b1mango** [comment]
基本相同😍

**Reply by rxliuli** [reply]
唉，不过吾辈已经决定要入手 mac m2 了，以后只有游戏会用 windows 比较多吧 (*/ω＼*)
